File: votação.py
n_eleitores = int(input("Quantas pessoas vão votar? "))
votos = {1: 0, 2: 0, 3: 0}

for i in range(n_eleitores):
    while True:
        voto = int(input(f"Voto {i+1} (Candidatos 1, 2 ou 3): "))
        if voto in [1, 2, 3]:
            votos[voto] += 1
            break
        else:
            print("Voto inválido! Tente novamente.")

print("-" * 20)
print(f"Candidato 1: {votos[1]} votos")
print(f"Candidato 2: {votos[2]} votos")
print(f"Candidato 3: {votos[3]} votos")

vencedor = max(votos, key=votos.get)
print(f"O vencedor é o Candidato {vencedor}!")

File: votação.c
#include <stdio.h>

int main() {
    int votantes, voto, i;
    int cand1 = 0, cand2 = 0, cand3 = 0;

    printf("Quantas pessoas vao votar? ");
    scanf("%d", &votantes);

    for (i = 0; i < votantes; i++) {
        do {
            printf("Eleitor %d - Vote (1, 2 ou 3): ", i + 1);
            scanf("%d", &voto);

            if (voto < 1 || voto > 3) {
                printf("Voto invalido! Tente novamente.\n");
            }
        } while (voto < 1 || voto > 3);

        if (voto == 1) cand1++;
        else if (voto == 2) cand2++;
        else if (voto == 3) cand3++;
    }

    printf("\n=== RESULTADO DA VOTACAO ===\n");
    printf("Candidato 1: %d votos\n", cand1);
    printf("Candidato 2: %d votos\n", cand2);
    printf("Candidato 3: %d votos\n", cand3);

    if (cand1 > cand2 && cand1 > cand3)
        printf("VENCEDOR: Candidato 1\n");
    else if (cand2 > cand1 && cand2 > cand3)
        printf("VENCEDOR: Candidato 2\n");
    else if (cand3 > cand1 && cand3 > cand2)
        printf("VENCEDOR: Candidato 3\n");
    else
        printf("EMPATE!\n");

    return 0;
}